### ISSUE TITLE
Lookup Tables: update routine

### ISSUE THREAD

**Issue by claudiosanches** [woocommerce/wc-admin#430]
There should be an update routine that populates all of the lookup tables when WC is upgraded.
Blocked until all the lookup tables are complete.

**Comment by timmyc** [comment]
@claudiosanches so just confirming the scope here - you are saying we need to hook into the the "Database update" button to trigger the lookup table build routine?

**Comment by claudiosanches** [comment]
@timmyc no, this is a upgrade routine, like updating from WooCommerce 3.5 to 3.6.
